SConstruct

```diff
@@ -104,6 +104,11 @@ def set_defaults(env, daos_version):
               action='store_true',
               default=False,
               help='Preprocess selected files for profiling')
+    AddOption('--stack_mmap',
```

Comment:
Two comments.   

This wants to be sticky so rather than making it an option, make it a variable.   See site_scons/prereq_tools/base.py.   Search add_opts (e.g. MPI_PKG, PREFIX, etc)

Also, the default should be enabled.  If you do that, you don't need to update daos.spec